### ISSUE TITLE
Desire the option to specify minimum angle above horizon

### ISSUE THREAD

**Issue by JamesSBriscoe** [dylan906/SatVisibility#3]
requires additional optional argument to vis_history and visibility_func functions

**Comment by dylan906** [comment]
Good idea! This sort of exists with the `hg` argument (extra height restriction above the planet surface). I think having both the 'extra height' and 'minimum angle' options for arguments would be best. It would just require deconfliction if values for both args are provided. Is there a use case for providing both hg and minimum angle? Need to decide whether or not to allow inputs for both.
